### PR TITLE
Some cosmetic modification and clarifications to the README.md file

### PR DESCRIPTION
**Pull request by arezk**
Hi , 
I have made some minor modifications and some extra clarifications to the README.md guide file (mostly cosmetic). Please review the modifications and let me know of any question. 
Thank you

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,19 +92,19 @@ This concludes the Cockpit section.
...
-- editing a topology by adding nodes and links. Nodes can be dragged from the palette on the right and then dropped onto the canvas. Links can be added by right-clicking a node and selecting "Add link"
-- configurations can be edited on "wiped" nodes
-- nodes can be interacted with via the console or VNC tab (where available)
-- on links, traffic conditions like delay or loss can be dialed in or a packet capture can be started
-- annotations (ellipses, rectangles, lines, arrows, text) can be added and edited
-- various display options can be toggled
+- Editing a topology by adding nodes and links. Nodes can be dragged from the palette on the right and then dropped onto the canvas. Links can be added by right-clicking a node and selecting "Add link"
+- Configurations can be edited on "wiped" nodes
+- Nodes can be interacted with via the console or VNC tab (where available)
+- Traffic conditions on links like delay or packer loss can be invoked or a packet capture can be started
```
Comment (rschmied) [inline README.md]: packet

File: README.md [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 Cisco Modeling Labs (CML) is the go-to tool for the simulation of virtual Cisco network devices and beyond. In this workshop, we’re going to cover the product from A to Z with a special focus on automation. In addition, we’re going to show how to extend the platform by adding Kali Linux.
 
-[TOC]
```
Comment (rschmied) [inline README.md]: should be kept -- it creates the table of content in the PDF

File: README.md [hunk 3]
```diff
@@ -365,7 +366,7 @@ This concludes the virl-utils section.
 
 ### Breakout tool
 
-The breakout tool allows to use native tools like PuTTY or VNC clients to connect to node consoles or graphical user interfaces.
+The breakout tool allows totive tools like PuTTY or VNC clients to connect to node consoles or graphical user interfaces.
```
Comment (rschmied) [inline README.md]: not sure if this is right here...? "totive"? :smile:

File: README.md [hunk 4]
```diff
@@ -396,17 +397,18 @@ Next, we need to verify the configuration. If nothing has been pre-configured th
...
+- Edit the created `labs.yaml` file and set the `enabled` flag to `true` for the labs you want to use. Note that if multiple labs are enabled, you need to ensure that there's no conflict in the ports being used.
   ![image-20230706114455254](./resources/image-20230706114455254.png)
 
-- run "run" -- `breakout run` to actually run the tool and make the ports available
+- Type `breakout run` to actually run the tool and make the ports available
   ![image-20230706114544952](./resources/image-20230706114544952.png)
 
-- use your native clients to connect to the ports displayed, there's PuTTY and UltraVNC available
+- use your native clients to connect to the ports displayed, there's PuTTY and UltraVNC available. 
+> **Note** to access `serial0` on the machine you need to use `telnet` to the defined port above
```
Comment (rschmied) [inline README.md]: Need to leave an empty line, i think, before the "Note" line. good call on the telnet. maybe specify that "when using PuTTY, ensure that the connection type is Telnet, not SSH"

**Comment by rschmied** [comment]
thanks @arezk -- I left a few comments that should to be addressed before I can merge this!

File: README.md (hunk 2 above)
Comment (arezk) [reply README.md]: Noted thanks

File: README.md (hunk 1 above)
Comment (arezk) [reply README.md]: will be corrected

File: README.md (hunk 3 above)
Comment (arezk) [reply README.md]: will be corrected -- backspace stuck :D

File: README.md (hunk 4 above)
Comment (arezk) [reply README.md]: I modified it that .. will be reflected in the new commit

File: README.md (hunk 2 above)
Comment (rschmied) [reply README.md]: now you added the `-` in front of the [TOC] -- needs to go.

**Review by rschmied — COMMENTED** [review]
Please remove the .idea directory from the commit. need to do `git rm --cached -r .idea`.

**Comment by rschmied** [comment]
OK -- this looks good now, I am going to merge this. Thanks for the help with this!

**Pull request merged**